### PR TITLE
Update the UMA SDK version for the demo vasp

### PR DESCRIPTION
**Pull request by jklein24**
Update the UMA SDK version for the demo vasp

Fix demo vasp localhost formats too

### CODE REVIEW

**Comment by jklein24** [comment]
Current dependencies on/for this PR:
* `develop`
  * **PR #139** <a href="https://app.graphite.dev/github/pr/lightsparkdev/kotlin-sdk/139?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/lightsparkdev/kotlin-sdk/139?utm_source=stack-comment).

File: umaserverdemo/src/main/kotlin/com/lightspark/plugins/HTTP.kt [hunk 1]
```diff
@@ -20,6 +22,8 @@ fun Application.configureHTTP() {
             Json {
                 prettyPrint = true
                 isLenient = true
+                ignoreUnknownKeys = true
+//                explicitNulls = false
```
Comment (mhrheaume) [inline umaserverdemo/src/main/kotlin/com/lightspark/plugins/HTTP.kt:26]: Are you intentionally committing this? Or just something you were messing around with locally?

**Review by mhrheaume — APPROVED** [review]

**Comment by jklein24** [comment]
### Merge activity

* **Jan 19, 2:05 PM**: @jklein24 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/lightsparkdev/kotlin-sdk/139).

File: umaserverdemo/src/main/kotlin/com/lightspark/plugins/HTTP.kt (hunk 1 above)
Comment (jklein24) [reply umaserverdemo/src/main/kotlin/com/lightspark/plugins/HTTP.kt:26]: Thanks for the catch! I meant to uncomment this before merging. Was testing something. Sending a followup now.

**Pull request merged**